app/views/questions/_answer.html.erb

```diff
@@ -29,10 +29,15 @@
     <% if current_user %>
       <%=  render :partial => "comments/edit", :locals => { title: "Edit answer", comment: answer, placeholder: "Help users by posting an answer to this question" } %>
     <% end %>
+    <% if current_user && current_user.uid == answer.author.uid %>
+      <% if answer.accepted %>
+        <div class="alert alert-success">Your answer was accepted! If it would be helpful, consider expanding on this answer by <a href="/post?template=activity,tags=response:<%= answer.nid %>">posting a step-by-step activity</a></div>
+      <% end %>
+    <% end %>
     <hr />
     <div style="padding: 0px 20px 10px 20px;">
-    <% if current_user && current_user.uid == answer.node.uid || answer.accepted %>
-      <a <% if current_user && current_user.uid == answer.node.uid %> href="/answers/accept/<%= answer.id %>" <% end %> data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm <% if answer.accepted %>btn-success<% else %>btn-default<% end%>"><% if answer.accepted %><p class="alert alert-success">Your answer was accepted! If it would be helpful, consider expanding on this answer by <a href="/post?template=activity,tags=response:<question-nid>">posting a step-by-step activity</a></p><% else %>Accept this answer<% end %></a>
+    <% if current_user && current_user.uid == answer.author.uid || answer.accepted %>
```

Comment:
I think this line should have answer.node.uid so that the person who asks the questions can choose to accept it.
What do you think?